src/index.ts

```diff
@@ -1,18 +1,24 @@
 #!/usr/bin/env node
 
-const { execSync } = require("child_process");
-const path = require("path");
-const Discord = require("discord.js");
-const fs = require("fs-extra");
-const prompts = require("prompts");
-const validatePackageName = require("validate-npm-package-name");
+import { execSync } from "child_process";
+import { Client } from "discord.js";
+import fs from "fs-extra";
+import path from "path";
+import prompts from "prompts";
 
-const appDirectory = path.join(__dirname, "app");
-const appPackage = require(path.join(appDirectory, "package.json"));
-const appToken = { token: "DISCORD_BOT_TOKEN_PLACEHOLDER" };
+import interfaces from "./declarations/interfaces";
+import types from "./declarations/types";
 
-const utilityPackage = require(path.join(__dirname, "package.json"));
-const utilityNameAndVersion = `${utilityPackage.name} v${utilityPackage.version}`;
+const validatePackageName: interfaces.ValidationFunction = require("validate-npm-package-name");
+
+const appDirectory: string = path.join(__dirname, "../app");
+const appPackage: types.Package = require(path.resolve(
+  path.join(appDirectory, "package.json")
+));
+const appToken: string = "DISCORD_BOT_TOKEN_PLACEHOLDER";
+
+const utilityPackage: types.Package = require(path.join("../", "package.json"));
```

Comment:
`require(path.join(__dirname, "../package.json"));`

similar usage of `..` to go back while starting from `__dirname` like for `appDirectory `